index.html

```diff
@@ -61,7 +61,7 @@
 
   <section id="getting-started">
     <aside>
-    <h3>Alright. How do I get started?</h3>
```

Comment:
Actually, "all right" is perfectly all right here -- while it does have the sense you mentioned, it is also an idiom roughly synonymous with "OK". See any good dictionary (e.g. http://dictionary.reference.com/browse/all+right ).